apps/client/sttr_build/index.js

```diff
@@ -55,6 +55,15 @@ function checkStatus(res) {
   return res;
 }
 
+/**
+ * Preprocessors are used to change the incoming xml. The xml
+ * is not yet parsed, so they are basic find replace functions.
+ * The output of every preprocessor is passed as input to the next.
+ */
+const preprocessors = [
+  (str) => str.replace(/ 14\s?020/g, " [14 020](tel:14020)"), // Fix phone-number
```

Comment:
This comment can be updated to something like "Replace phone number with Link"

Reply:
Check